mypy/nodes.py

```diff
@@ -390,6 +405,10 @@ def set_line(self, target: Union[Token, Node, int]) -> Node:
             self.initialization_statement.set_line(self.line)
             self.initialization_statement.lvalues[0].set_line(self.line)
 
+    def set_column(self, target: Union[Token, Node, int]) -> Node:
+        super().set_column(target)
+        return self
```

Comment:
I find `return self` an anti-pattern that encourages unneeded cleverness and confuses side effects with functions.

Maybe `set_line()` should get an optional `column` method? That would also beautifully allow you to call it with a Token or Node and copy both line and column from there.